cypress/integration/trakt/userLists.spec.js

```diff
@@ -0,0 +1,214 @@
+/// <reference types="cypress" />
+describe('02: Trakt API User list ', () => {
+	describe('02_001: Create, Update and Delete operations with users lists', () => {
+		before(function () {
+			cy.getAccessToken()
+		})
+
+		it('02_001_001: Should delete all User Lists', () => {
```

Comment:
You need to assert this here, unless you do it in the command it self. But for the sake of consistency, assertions should be done in it() blocks, and the commands should return the result on which it is possible to make an assertion.

Reply:
For command 'deleteAllLists' , I can't return (I tried but it doesn't work) the status value 204 for the delete method, because the delete method is inside the get method and I can only return the status 200, and make an assertion for that status in it () block